content_scripts/localize.js

```diff
@@ -0,0 +1,8 @@
+window.addEventListener('load', () => {
```

Comment:
This seems only for those html files in `pages/`, so we may move it to `pages/localize.js`.